src/utils/roomUtil.ts

```diff
@@ -2,6 +2,7 @@ import { DecisionRoomHandler } from '../roomHandlers/decisionRoomHandler';
 import { ExampleRoom1Handler } from '../roomHandlers/exampleRoom1Handler';
 import { ExampleRoom2Handler } from '../roomHandlers/exampleRoom2Handler';
 import { ExampleRoom3Handler } from '../roomHandlers/exampleRoom3Handler';
+import { FishingRoomHandler } from '../roomHandlers/fishingRoomHandler';
```

Comment:
No change requested: I see in your second commit you add this to fix the type issue with the room util, but `fishingRoomHandler` wasn't committed so it's a bit weird.